compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -231,6 +232,13 @@ def containerized_launch_cmd(self) -> str:
                 command=launch_cmd,
                 options=self.container_cmd_options or "",
             )
+        if self.container_type == "podman":
+            launch_cmd = PODMAN_CMD_TEMPLATE.format(
```

Comment:
Given these branches are mutually exclusive (e.g., `self.container_type` is static for the duration of this method), this isn't wrong per se, but it does break the `elif` chain set by the surrounding code.  (Yes, please use `elif`.)